blockchain/difficulty.go

```diff
@@ -286,7 +287,8 @@ func (b *BlockChain) calcNextRequiredDifficulty(lastNode *blockNode, newBlockTim
 
 	// Limit the amount of adjustment that can occur to the previous
 	// difficulty.
-	actualTimespan := lastNode.timestamp.UnixNano() - firstNode.timestamp.UnixNano()
```

Comment:
Oh yeah, I should have been less lazy and checked the definition of `minRetargetTimespan`, but I go through the changes via e-mail in GitHub, instead of checking them out.

No problem, I really think this project is great. I should try to make more time to contribute, though :)